google/cloud/bigquery/_tqdm_helpers.py

```diff
@@ -15,13 +15,15 @@
 """Shared helper functions for tqdm progress bar."""
 
 import concurrent.futures
+import sys
 import time
 import typing
 from typing import Optional
 import warnings
 
 try:
     import tqdm  # type: ignore
+    import tqdm.notebook as notebook  # type: ignore
```

Comment:
This additional import test is to ensure we're dealing with a modern tqdm installation?  We don't test for tqdm_gui, should we?  Do we use it in a meaningful way?

Reply:
Right, line 26 is the correct import